### DISCUSSION TITLE
What's next, an infinitely scrolling 9:16 feed with music and subway surfers gameplay?

### DISCUSSION THREAD

**Discussion by matt-mekha** [community/community#54406] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

For real though, one issue I had with it that could be improved upon is that I contributed to this open source repository once, and now this one user (that merged my PR I think) is front and center of my feed despite no longer being relevant to me. I use the feed to keep up with my friends, and I'm already following my friends.

I think the ideal solution is to just put it back to how it was.

**Comment by JennPlothow** [comment]
### **### [raw.githubusercontent.com_community_community_1feab22b89c2a6ea58ff3fe27abc7e20c10d86f4_CODE_OF_CONDUCT.md.txt](https://github.com/community/community/files/12036454/raw.githubusercontent.com_community_community_1feab22b89c2a6ea58ff3fe27abc7e20c10d86f4_CODE_OF_CONDUCT.md.txt)**
